10/umbraco-cms/reference/management/models/contenttype.md

```diff
@@ -162,7 +162,7 @@ return contentType.Icon;
 
 ### .Id
 
-Gets the unique `ContentType` Id as a `Int`, this ID is based on a Database identity field, and is therefore not safe to reference in code which are moved between different instances, use Key instead.
+Gets the unique `ContentType` Id as an `Int`, this ID is based on a Database identity field. It's not safe for code referencing across different instances, use Key instead.
```

Comment:
To keep the same wording as per other docs: https://github.com/umbraco/UmbracoDocs/pull/5674/files (same goes for v12 and v13 as well)
```suggestion
Gets the unique `ContentType` Id as an `Int`. The ID, derived from a database identity field, isn't safe for code references as they are moved across instances. Therefore it's recommended to use `Key` instead.
```

Reply:
@alina-tincas Good point! 😄